### ISSUE TITLE
openocd for windows mass_erase not working

### ISSUE THREAD

**Issue by rananna** [OpenSourceEBike/TSDZ2_wireless#108]
We need a fix for openocd for windows.
Mass_erase is not working and it is needed for the s340 softdevice.

**Comment by 4var1** [comment]
i don't understand - it works fine.  I do it all the time - is in the openocd.cfg i posted.

I took this reply from filippods to mean it works all ways when powered with 5v...

Sorry, i've accidentally renamed it, now WORKS (**Verify OK**) :i've flashed bootloader with openocd (and openocd.cfg)... The nrf52 must be powered on with 5v, in fact when i power on with 3.3v i can't flash with openocd.cfg in the folder.

**Comment by rananna** [comment]
Yes, he had correct verification many times under windows, but the boot loader would not flash. Go down two more posts:

"OK, now what happens when you flash the 0.6.2 firmware using the bootloader?
(keep the board powered with 5V.)

The leds still not blinking and the device not appear in bt list"

And then another post just under that:
"I've completed the linux procedure, now all works fine, i can connect to TSDZ2."

He can flash successfully under Linux.

Anyway, he has gone silent. Maybe nobody else will have these issues with windows.

**Comment by 4var1** [comment]
Ok - i think we see who else has issues, if anyone. I did the original bootloader flash and OTA update fine from Windows.

**Comment by rananna** [comment]
I will keep the issue open for now.
I read on other forums that out of date stlink firmware can cause an issue with mass_erase under windows.
Maybe that was his issue?
Anyway, I just posted a question for you on the forum - could you please have a look?
